### PR TITLE
vmui: change logic autocomplete

### PR DESCRIPTION
**Pull request by Loori-R**
- Show autocomplete if query string contains one word;
- Fixed detect key "command" for Mac OS;
- Fixed select value by clicking mouse.

### CODE REVIEW

File: app/vmui/packages/vmui/src/components/Home/Configurator/Query/QueryEditor.tsx [hunk 1]
```diff
@@ -86,24 +87,33 @@ const QueryEditor: FC<QueryEditorProps> = ({
     if (target?.scrollIntoView) target.scrollIntoView({block: "center"});
   }, [focusOption]);
 
-  return <Box ref={autocompleteAnchorEl}>
+  return <Box ref={autocompleteAnchorEl} sx={{position: "relative", zIndex: focusField ? 2 : 1}}>
```
Comment (valyala) [inline app/vmui/packages/vmui/src/components/Home/Configurator/Query/QueryEditor.tsx:90]: It looks like this leads to the following bug:

vmui_bug

**Review by valyala — CHANGES_REQUESTED** [review]

File: app/vmui/packages/vmui/src/components/Home/Configurator/Query/QueryEditor.tsx [hunk 2]
```diff
@@ -38,13 +38,14 @@ const QueryEditor: FC<QueryEditorProps> = ({
   const wrapperEl = useRef<HTMLUListElement>(null);
 
   const openAutocomplete = useMemo(() => {
-    return !(!autocomplete || downMetaKeys.length || query.length < 2 || !focusField);
+    const words = query.split(/_|\.|\s+|=|\{/).length;
```
Comment (valyala) [inline app/vmui/packages/vmui/src/components/Home/Configurator/Query/QueryEditor.tsx:41]: The `_` and `.` can present in metric name. See the full list of characters, which can present in metric name at  https://prometheus.io/docs/concepts/data_model/#metric-names-and-labels . VictoriaMetrics additionally supports also `.` character, which is frequently used in [Graphite metrics](https://stackoverflow.com/questions/17503994/correct-use-of-graphite-metric-names), which are [accepted by VictoriaMetrics](https://docs.victoriametrics.com/#how-to-send-data-from-graphite-compatible-agents-such-as-statsd).

File: app/vmui/packages/vmui/src/components/Home/Configurator/Query/QueryEditor.tsx (hunk 1 above)
Comment (Loori-R) [reply app/vmui/packages/vmui/src/components/Home/Configurator/Query/QueryEditor.tsx]: Fixed

File: app/vmui/packages/vmui/src/components/Home/Configurator/Query/QueryEditor.tsx (hunk 2 above)
Comment (Loori-R) [reply app/vmui/packages/vmui/src/components/Home/Configurator/Query/QueryEditor.tsx]: Changed the logic

**Review by valyala — APPROVED** [review]

**Pull request merged**